### ISSUE TITLE
[4.0] Nested tabs navigation issue

### ISSUE THREAD

**Issue by tassosm** [joomla/joomla-cms#20627]
It looks like there's a conflict with the navigation of nested tabsets.

### Steps to reproduce the issue
```php
echo JHtml::_('bootstrap.startTabSet', 'outerTab', array('active' => 'x1'));

echo JHtml::_('bootstrap.addTab', 'outerTab', 'x1', 'x1');
echo 'Outer Tab 1';
echo JHtml::_('bootstrap.endTab');

echo JHtml::_('bootstrap.addTab', 'outerTab', 'x2', 'x2');
echo 'Outer Tab 2';

// Nested tabset
echo JHtml::_('bootstrap.startTabSet', 'innerTab', array('active' => 'y1'));

echo JHtml::_('bootstrap.addTab', 'innerTab', 'y1', 'y1');
echo 'Inner Tab 1';
echo JHtml::_('bootstrap.endTab');

echo JHtml::_('bootstrap.addTab', 'innerTab', 'y2', 'y2');
echo 'Inner Tab 2';
echo JHtml::_('bootstrap.endTab');

echo JHtml::_('bootstrap.endTabSet');
// End of nested tabset

echo JHtml::_('bootstrap.endTab');
echo JHtml::_('bootstrap.endTabSet');
```

### Expected result
![tabs_issue_j4_ok](https://user-images.githubusercontent.com/6807469/40710124-8049af56-6400-11e8-9b36-9c036e4610d5.gif)

### Actual result
![tabs_issue_j4](https://user-images.githubusercontent.com/6807469/40710102-705710ca-6400-11e8-89f0-87b5ece45bc1.gif)

### System information (as much as possible)
Joomla! 4.0.0-alpha4-dev
7.1.11-1+ubuntu14.04.1+deb.sury.org+1 

### Additional comments
No issue on Joomla 3.8.8

![tabs_issue_j3](https://user-images.githubusercontent.com/6807469/40710197-b0115bd0-6400-11e8-88f3-1a354664b93c.gif)

**Comment by PhilETaylor** [comment]
I did a LOT of work on nested tab state in Joomla 3.x, however Im guessing that this is something brand new in Joomla 4 thats not using what I did...

**Comment by ghost** [comment]
@Taatshi can you please look if this is an Issue on latest Nightly Build?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20627">issues.joomla.org/tracker/joomla-cms/20627</a>.</sub>

**Comment by SharkyKZ** [comment]
Test PR #25420 please.

**Comment by alikon** [comment]
closed for #25420<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20627">issues.joomla.org/tracker/joomla-cms/20627</a>.</sub>
